packages/p2p-v2/src/pages/my-profile/screens/PaymentMethods/PaymentMethodsList/AddNewButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Button } from '@deriv-com/ui/dist/components/Button';
+
+type TAddNewButtonProps = {
+    isMobile: boolean;
+    onAdd: () => void;
+};
+
+/**
+ * @component This component is used to display the add new button
+ * @param isMobile - Whether the current device is mobile or not
+ * @param onAdd - The function to be called when the button is clicked
+ * @returns {JSX.Element}
+ * @example <AddNewButton isMobile={isMobile} onAdd={onAdd} />
+ * **/
+const AddNewButton = ({ isMobile, onAdd }: TAddNewButtonProps) => {
+    return (
+        <Button isFullWidth={isMobile} onClick={() => onAdd()} size='lg'>
```

Comment:
`onClick={onAdd}`

Reply:
If this is added then `onAdd` would need to be typed with accepting `EventHandler` right? Since `onAdd` is not using the event I think this is fine its just syntactic sugar, otherwise it would get an unused` event` argument error

Reply:
@nada-deriv, also the `onAdd` function expects an optional argument as seen below, therefore calling it like that would send an invalid argument (the `event objeect`
![nada](https://github.com/binary-com/deriv-app/assets/104334373/83d9c0c1-0cce-423e-b0b4-f37b594091bc)
) to the function